packages/teams-js/src/public/media.ts

```diff
@@ -739,82 +687,46 @@ export namespace media {
 
   /**
    * Scan Barcode/QRcode using camera
-   *
    * @remarks
    * Note: For desktop and web, this API is not supported. Callback will be resolved with ErrorCode.NotSupported.
    *
-   * @param config - optional input configuration to customize the barcode scanning experience
-   * @returns A promise resolved with the barcode data or rejected with an @see SdkError
-   */
-  export function scanBarCode(config?: BarCodeConfig): Promise<string>;
-  /**
-   * Scan Barcode/QRcode using camera
-   *
-   * @remarks
-   * Note: For desktop and web, this API is not supported. Callback will be resolved with ErrorCode.NotSupported.
-   *
-   * @deprecated
-   * As of 2.0.0-beta.1, please use {@link media.scanBarCode media.scanBarCode(config?: BarCodeConfig): Promise\<string\>} instead.
-   *
    * @param callback - callback to invoke after scanning the barcode
    * @param config - optional input configuration to customize the barcode scanning experience
    */
-  export function scanBarCode(callback: (error: SdkError, decodedText: string) => void, config?: BarCodeConfig);
```

Comment:
Not for this change, but probably we should update this documentation to point to updated barcode capability when that's ready.

Reply:
Yeah, when we write the new bar code, we can update the documentation here.